### ISSUE TITLE
Unable to complete boot process after routine upgrade from 20.1.7 to 20.7.8

### ISSUE THREAD

**Issue by averyfreeman** [opnsense/update#67]
I also left a message about this here:

https://forum.opnsense.org/index.php?topic=21116.0

I'm not sure if it has to do with the update tools or core.

steps:

1. Saw there was an update in `system -> firmware -> updates` 
2. Clicked "install"
3. System rebooted after update completed
4. After reboot, system could not complete boot process

Expected behavior: 

System complete boot process, land in `opnsense-shell`

Result:

System could not complete boot.  

How'd you resolve your issue?

I rebooted manually and loaded `/boot//kernel.old`, was able to complete boot process then.

Logs:

This is my `kernel.old`:

```
root@gateway:/var/run # uname -a

FreeBSD gateway.webtool.space 12.2-RELEASE-p2 FreeBSD 12.2-RELEASE-p2 #0 r369071M: Wed Jan 20 13:32:25 PST 2021     avery@fabby.webtool.space:/usr/obj/usr/src/amd64.amd64/sys/CALOMEL  amd64
```

Tail of `pkg.log`:

```
root@gateway:/var/log # tail pkg.log

Jan 18 18:11:52 gateway pkg[42852]: pkg upgraded: 1.12.0_1 -> 1.15.10_1
Jan 18 18:11:54 gateway pkg[21286]: py37-speedtest-cli-2.1.2 installed
Jan 18 20:43:44 gateway pkg[7559]: os-theme-cicada-1.25 installed
Jan 18 20:44:00 gateway pkg[70015]: os-theme-vicuna-1.1 installed
Jan 26 19:56:26 gateway pkg-static[78977]: os-dyndns upgraded: 1.22 -> 1.23
Jan 26 19:56:38 gateway pkg-static[78977]: opnsense upgraded: 20.7 -> 20.7.8
Jan 26 19:56:38 gateway pkg-static[72426]: py37-asn1crypto-1.3.0 de
```

`dmesg` from boot that did NOT complete:

```
root@gateway:/var/run # less dmesg.boot

---<<BOOT>>---
Copyright (c) 1992-2020 The FreeBSD Project.
Copyright (c) 1979, 1980, 1983, 1986, 1988, 1989, 1991, 1992, 1993, 1994
        The Regents of the University of California. All rights reserved.
FreeBSD is a registered trademark of The FreeBSD Foundation.
FreeBSD 12.2-RELEASE-p2 #0 r369071M: Wed Jan 20 13:32:25 PST 2021
    avery@fabby.webtool.space:/usr/obj/usr/src/amd64.amd64/sys/CALOMEL amd64
FreeBSD clang version 10.0.1 (git@github.com:llvm/llvm-project.git llvmorg-10.0.1-0-gef32c611aa2)
VT(efifb): resolution 1024x768
CPU: Intel(R) Celeron(R) CPU  J1800  @ 2.41GHz (2416.73-MHz K8-class CPU)
  Origin="GenuineIntel"  Id=0x30673  Family=0x6  Model=0x37  Stepping=3
  Features=0xbfebfbff<FPU,VME,DE,PSE,TSC,MSR,PAE,MCE,CX8,APIC,SEP,MTRR,PGE,MCA,CMOV,PAT,PSE36,CLFLUSH,DTS,ACPI,MMX,FXSR,SSE,SSE2,SS,HTT,TM,PBE>
  Features2=0x41d8e3bf<SSE3,PCLMULQDQ,DTES64,MON,DS_CPL,VMX,EST,TM2,SSSE3,CX16,xTPR,PDCM,SSE4.1,SSE4.2,MOVBE,POPCNT,TSCDLT,RDRAND>
  AMD Features=0x28100800<SYSCALL,NX,RDTSCP,LM>
  AMD Features2=0x101<LAHF,Prefetch>
  Structured Extended Features=0x2282<TSCADJ,SMEP,ERMS,NFPUSG>
  VT-x: PAT,HLT,MTF,PAUSE,EPT,UG,VPID
  TSC: P-state invariant, performance statistics
real memory  = 8589934592 (8192 MB)
avail memory = 8145580032 (7768 MB)
Event timer "LAPIC" quality 600
ACPI APIC Table: <LENOVO TC-03   >
WARNING: L1 data cache covers fewer APIC IDs than a core (0 < 1)
FreeBSD/SMP: Multiprocessor System Detected: 2 CPUs
FreeBSD/SMP: 1 package(s) x 2 core(s)
random: unblocking device.
Firmware Warning (ACPI): 32/64X length mismatch in FADT/Gpe0Block: 128/32 (20200430/tbfadt-748)
ioapic0 <Version 2.0> irqs 0-86 on motherboard
Launching APs: 1
Timecounter "TSC-low" frequency 1208363314 Hz quality 1000
random: entropy device external interface
kbd1 at kbdmux0
000.000023 [4336] netmap_init               netmap: loaded module
[ath_hal] loaded
module_register_init: MOD_LOAD (vesa, 0xffffffff81117e40, 0) error 19
random: registering fast source Intel Secure Key RNG
random: fast provider: "Intel Secure Key RNG"
nexus0
kernel trap 12 with interrupts disabled
kernel trap 12 with interrupts disabled
cryptosoft0: <software crypto> on motherboard
acpi0: <LENOVO TC-03> on motherboard
acpi0: Power Button (fixed)
unknown: I/O range not supported
cpu0: <ACPI CPU> on acpi0
atrtc0: <AT realtime clock> port 0x70-0x77 on acpi0
atrtc0: Warning: Couldn't map I/O.
atrtc0: registered as a time-of-day clock, resolution 1.000000s
Event timer "RTC" frequency 32768 Hz quality 0
hpet0: <High Precision Event Timer> iomem 0xfed00000-0xfed003ff irq 8 on acpi0
Timecounter "HPET" frequency 14318180 Hz quality 950
Event timer "HPET" frequency 14318180 Hz quality 450
Event timer "HPET1" frequency 14318180 Hz quality 440
Event timer "HPET2" frequency 14318180 Hz quality 440
attimer0: <AT timer> port 0x40-0x43,0x50-0x53 irq 0 on acpi0
Timecounter "i8254" frequency 1193182 Hz quality 0
Event timer "i8254" frequency 1193182 Hz quality 100
Timecounter "ACPI-safe" frequency 3579545 Hz quality 850
acpi_timer0: <24-bit timer at 3.579545MHz> port 0x408-0x40b on acpi0
pcib0: <ACPI Host-PCI bridge> port 0xcf8-0xcff on acpi0
pcib0: Length mismatch for 3 range: 10b15fff vs 10b16000
pci0: <ACPI PCI bus> on pcib0
vgapci0: <VGA-compatible display> port 0xf080-0xf087 mem 0xb0000000-0xb03fffff,0xa0000000-0xafffffff irq 16 at device 2.0 on pci0
vgapci0: Boot video device
ahci0: <AHCI SATA controller> port 0xf070-0xf077,0xf060-0xf063,0xf050-0xf057,0xf040-0xf043,0xf020-0xf03f mem 0xb0b15000-0xb0b157ff irq 19 at device 19.0 on pci0
ahci0: AHCI v1.30 with 2 3Gbps ports, Port Multiplier not supported
ahcich0: <AHCI channel> at channel 0 on ahci0
ahcich1: <AHCI channel> at channel 1 on ahci0
xhci0: <Intel BayTrail USB 3.0 controller> mem 0xb0b00000-0xb0b0ffff irq 20 at device 20.0 on pci0
xhci0: 32 bytes context size, 64-bit DMA
xhci0: Port routing mask set to 0xffffffff
usbus0 on xhci0
usbus0: 5.0Gbps Super Speed USB v3.0
pci0: <encrypt/decrypt> at device 26.0 (no driver attached)
pcib1: <ACPI PCI-PCI bridge> at device 28.0 on pci0
pcib1: [GIANT-LOCKED]
pci1: <ACPI PCI bus> on pcib1
pcib2: <ACPI PCI-PCI bridge> at device 0.0 on pci1
pci2: <ACPI PCI bus> on pcib2
pcib3: <PCI-PCI bridge> at device 2.0 on pci2
pci3: <PCI bus> on pcib3
em0: <Intel(R) PRO/1000 Network Connection> port 0xd020-0xd03f mem 0xb0920000-0xb093ffff,0xb0880000-0xb08fffff at device 0.0 on pci3
em0: Using 1024 TX descriptors and 1024 RX descriptors
em0: Using an MSI interrupt
em0: Ethernet address: 00:26:55:d6:b1:15
em0: netmap queues/slots: TX 1/1024, RX 1/1024
em1: <Intel(R) PRO/1000 Network Connection> port 0xd000-0xd01f mem 0xb0900000-0xb091ffff,0xb0800000-0xb087ffff at device 0.1 on pci3
em1: Using 1024 TX descriptors and 1024 RX descriptors
em1: Using an MSI interrupt
em1: Ethernet address: 00:26:55:d6:b1:14
em1: netmap queues/slots: TX 1/1024, RX 1/1024
pcib4: <PCI-PCI bridge> at device 4.0 on pci2
pci4: <PCI bus> on pcib4
em2: <Intel(R) PRO/1000 Network Connection> port 0xc020-0xc03f mem 0xb0720000-0xb073ffff,0xb0680000-0xb06fffff at device 0.0 on pci4
em2: Using 1024 TX descriptors and 1024 RX descriptors
em2: Using an MSI interrupt
em2: Ethernet address: 00:26:55:d6:b1:17
em2: netmap queues/slots: TX 1/1024, RX 1/1024
em3: <Intel(R) PRO/1000 Network Connection> port 0xc000-0xc01f mem 0xb0700000-0xb071ffff,0xb0600000-0xb067ffff at device 0.1 on pci4
em3: Using 1024 TX descriptors and 1024 RX descriptors
em3: Using an MSI interrupt
em3: Ethernet address: 00:26:55:d6:b1:16
em3: netmap queues/slots: TX 1/1024, RX 1/1024
pcib5: <ACPI PCI-PCI bridge> at device 28.2 on pci0
pcib5: [GIANT-LOCKED]
pci5: <ACPI PCI bus> on pcib5
re0: <RealTek 810xE PCIe 10/100baseTX> port 0xe000-0xe0ff mem 0xb0a04000-0xb0a04fff,0xb0a00000-0xb0a03fff irq 18 at device 0.0 on pci5
re0: Using 1 MSI-X message
re0: ASPM disabled
re0: Chip rev. 0x40800000
re0: MAC rev. 0x00400000
miibus0: <MII bus> on re0
rlphy0: <RTL8201E 10/100 media interface> PHY 1 on miibus0
rlphy0:  10baseT, 10baseT-FDX, 100baseTX, 100baseTX-FDX, auto, auto-flow
re0: Using defaults for TSO: 65518/35/2048
re0: Ethernet address: c0:3f:d5:93:3d:a7
re0: netmap queues/slots: TX 1/256, RX 1/256
pcib6: <ACPI PCI-PCI bridge> at device 28.3 on pci0
ada0: 22902MB (46905264 512 byte sectors)
ada1 at ahcich1 bus 0 scbus1 target 0 lun 0
ada1: <INTEL SSDMAEXC024G3H 9CV10379> ATA8-ACS SATA 2.x device
ada1: Serial Number CVHA3433003R024D
ada1: 300.000MB/s transfers (SATA 2.x, UDMA6, PIO 8192bytes)
ada1: Command Queueing enabled
ada1: 22902MB (46905264 512 byte sectors)
GEOM: ada0: the secondary GPT header is not in the last LBA.
GEOM: diskid/DISK-CVHA3433003W024D: the secondary GPT header is not in the last LBA.
GEOM: ada1: the secondary GPT header is not in the last LBA.
GEOM: diskid/DISK-CVHA3433003R024D: the secondary GPT header is not in the last LBA.
uhub0: 7 ports with 7 removable, self powered
Root mount waiting for: usbus0
ugen0.2: <vendor 0x05e3 USB2.0 Hub> at usbus0
uhub1 on uhub0
uhub1: <vendor 0x05e3 USB2.0 Hub, class 9/0, rev 2.00/32.98, addr 1> on usbus0
uhub1: MTT enabled
uhub1: 4 ports with 4 removable, self powered
Root mount waiting for: usbus0
ugen0.3: <vendor 0x04d9 USB Keyboard> at usbus0
ukbd0 on uhub1
ukbd0: <vendor 0x04d9 USB Keyboard, class 0/0, rev 1.10/3.10, addr 2> on usbus0
kbd2 at ukbd0
mountroot: waiting for device /dev/gpt/rootfs...
GEOM: diskid/DISK-CVHA3433003W024D: the secondary GPT header is not in the last LBA.
GEOM: diskid/DISK-CVHA3433003W024D: the secondary GPT header is not in the last LBA.
GEOM: diskid/DISK-CVHA3433003W024D: the secondary GPT header is not in the last LBA.
```

`dmesg` from boot that COULD boot (for reference):

```
root@gateway:/var/run # dmesg -a
---<<BOOT>>---
Copyright (c) 1992-2020 The FreeBSD Project.
Copyright (c) 1979, 1980, 1983, 1986, 1988, 1989, 1991, 1992, 1993, 1994
        The Regents of the University of California. All rights reserved.
FreeBSD is a registered trademark of The FreeBSD Foundation.
FreeBSD 12.2-RELEASE-p2 #0 r369071M: Wed Jan 20 13:32:25 PST 2021
    avery@fabby.webtool.space:/usr/obj/usr/src/amd64.amd64/sys/CALOMEL amd64
FreeBSD clang version 10.0.1 (git@github.com:llvm/llvm-project.git llvmorg-10.0.1-0-gef32c611aa2)
VT(efifb): resolution 1024x768
CPU: Intel(R) Celeron(R) CPU  J1800  @ 2.41GHz (2416.73-MHz K8-class CPU)
  Origin="GenuineIntel"  Id=0x30673  Family=0x6  Model=0x37  Stepping=3
  Features=0xbfebfbff<FPU,VME,DE,PSE,TSC,MSR,PAE,MCE,CX8,APIC,SEP,MTRR,PGE,MCA,CMOV,PAT,PSE36,CLFLUSH,DTS,ACPI,MMX,FXSR,SSE,SSE2,SS,HTT,TM,PBE>
  Features2=0x41d8e3bf<SSE3,PCLMULQDQ,DTES64,MON,DS_CPL,VMX,EST,TM2,SSSE3,CX16,xTPR,PDCM,SSE4.1,SSE4.2,MOVBE,POPCNT,TSCDLT,RDRAND>
  AMD Features=0x28100800<SYSCALL,NX,RDTSCP,LM>
  AMD Features2=0x101<LAHF,Prefetch>
  Structured Extended Features=0x2282<TSCADJ,SMEP,ERMS,NFPUSG>
  VT-x: PAT,HLT,MTF,PAUSE,EPT,UG,VPID
  TSC: P-state invariant, performance statistics
real memory  = 8589934592 (8192 MB)
avail memory = 8145580032 (7768 MB)
Event timer "LAPIC" quality 600
ACPI APIC Table: <LENOVO TC-03   >
WARNING: L1 data cache covers fewer APIC IDs than a core (0 < 1)
FreeBSD/SMP: Multiprocessor System Detected: 2 CPUs
FreeBSD/SMP: 1 package(s) x 2 core(s)
random: unblocking device.
Firmware Warning (ACPI): 32/64X length mismatch in FADT/Gpe0Block: 128/32 (20200430/tbfadt-748)
ioapic0 <Version 2.0> irqs 0-86 on motherboard
Launching APs: 1
Timecounter "TSC-low" frequency 1208363314 Hz quality 1000
random: entropy device external interface
kbd1 at kbdmux0
000.000023 [4336] netmap_init               netmap: loaded module
[ath_hal] loaded
module_register_init: MOD_LOAD (vesa, 0xffffffff81117e40, 0) error 19
random: registering fast source Intel Secure Key RNG
random: fast provider: "Intel Secure Key RNG"
nexus0
kernel trap 12 with interrupts disabled
kernel trap 12 with interrupts disabled
cryptosoft0: <software crypto> on motherboard
acpi0: <LENOVO TC-03> on motherboard
acpi0: Power Button (fixed)
unknown: I/O range not supported
cpu0: <ACPI CPU> on acpi0
atrtc0: <AT realtime clock> port 0x70-0x77 on acpi0
atrtc0: Warning: Couldn't map I/O.
atrtc0: registered as a time-of-day clock, resolution 1.000000s
Event timer "RTC" frequency 32768 Hz quality 0
hpet0: <High Precision Event Timer> iomem 0xfed00000-0xfed003ff irq 8 on acpi0
Timecounter "HPET" frequency 14318180 Hz quality 950
Event timer "HPET" frequency 14318180 Hz quality 450
Event timer "HPET1" frequency 14318180 Hz quality 440
Event timer "HPET2" frequency 14318180 Hz quality 440
attimer0: <AT timer> port 0x40-0x43,0x50-0x53 irq 0 on acpi0
Timecounter "i8254" frequency 1193182 Hz quality 0
Event timer "i8254" frequency 1193182 Hz quality 100
Timecounter "ACPI-safe" frequency 3579545 Hz quality 850
acpi_timer0: <24-bit timer at 3.579545MHz> port 0x408-0x40b on acpi0
pcib0: <ACPI Host-PCI bridge> port 0xcf8-0xcff on acpi0
pcib0: Length mismatch for 3 range: 10b15fff vs 10b16000
pci0: <ACPI PCI bus> on pcib0
vgapci0: <VGA-compatible display> port 0xf080-0xf087 mem 0xb0000000-0xb03fffff,0xa0000000-0xafffffff irq 16 at device 2.0 on pci0
vgapci0: Boot video device
ahci0: <AHCI SATA controller> port 0xf070-0xf077,0xf060-0xf063,0xf050-0xf057,0xf040-0xf043,0xf020-0xf03f mem 0xb0b15000-0xb0b157ff irq 19 at device 19.0 on pci0
ahci0: AHCI v1.30 with 2 3Gbps ports, Port Multiplier not supported
ahcich0: <AHCI channel> at channel 0 on ahci0
ahcich1: <AHCI channel> at channel 1 on ahci0
xhci0: <Intel BayTrail USB 3.0 controller> mem 0xb0b00000-0xb0b0ffff irq 20 at device 20.0 on pci0
xhci0: 32 bytes context size, 64-bit DMA
xhci0: Port routing mask set to 0xffffffff
usbus0 on xhci0
usbus0: 5.0Gbps Super Speed USB v3.0
pci0: <encrypt/decrypt> at device 26.0 (no driver attached)
pcib1: <ACPI PCI-PCI bridge> at device 28.0 on pci0
pcib1: [GIANT-LOCKED]
pci1: <ACPI PCI bus> on pcib1
pcib2: <ACPI PCI-PCI bridge> at device 0.0 on pci1
pci2: <ACPI PCI bus> on pcib2
pcib3: <PCI-PCI bridge> at device 2.0 on pci2
pci3: <PCI bus> on pcib3
em0: <Intel(R) PRO/1000 Network Connection> port 0xd020-0xd03f mem 0xb0920000-0xb093ffff,0xb0880000-0xb08fffff at device 0.0 on pci3
em0: Using 1024 TX descriptors and 1024 RX descriptors
em0: Using an MSI interrupt
em0: Ethernet address: 00:26:55:d6:b1:15
em0: netmap queues/slots: TX 1/1024, RX 1/1024
em1: <Intel(R) PRO/1000 Network Connection> port 0xd000-0xd01f mem 0xb0900000-0xb091ffff,0xb0800000-0xb087ffff at device 0.1 on pci3
em1: Using 1024 TX descriptors and 1024 RX descriptors
em1: Using an MSI interrupt
em1: Ethernet address: 00:26:55:d6:b1:14
em1: netmap queues/slots: TX 1/1024, RX 1/1024
pcib4: <PCI-PCI bridge> at device 4.0 on pci2
pci4: <PCI bus> on pcib4
em2: <Intel(R) PRO/1000 Network Connection> port 0xc020-0xc03f mem 0xb0720000-0xb073ffff,0xb0680000-0xb06fffff at device 0.0 on pci4
em2: Using 1024 TX descriptors and 1024 RX descriptors
em2: Using an MSI interrupt
em2: Ethernet address: 00:26:55:d6:b1:17
em2: netmap queues/slots: TX 1/1024, RX 1/1024
em3: <Intel(R) PRO/1000 Network Connection> port 0xc000-0xc01f mem 0xb0700000-0xb071ffff,0xb0600000-0xb067ffff at device 0.1 on pci4
em3: Using 1024 TX descriptors and 1024 RX descriptors
em3: Using an MSI interrupt
em3: Ethernet address: 00:26:55:d6:b1:16
em3: netmap queues/slots: TX 1/1024, RX 1/1024
pcib5: <ACPI PCI-PCI bridge> at device 28.2 on pci0
pcib5: [GIANT-LOCKED]
pci5: <ACPI PCI bus> on pcib5
re0: <RealTek 810xE PCIe 10/100baseTX> port 0xe000-0xe0ff mem 0xb0a04000-0xb0a04fff,0xb0a00000-0xb0a03fff irq 18 at device 0.0 on pci5
re0: Using 1 MSI-X message
re0: ASPM disabled
re0: Chip rev. 0x40800000
re0: MAC rev. 0x00400000
miibus0: <MII bus> on re0
rlphy0: <RTL8201E 10/100 media interface> PHY 1 on miibus0
rlphy0:  10baseT, 10baseT-FDX, 100baseTX, 100baseTX-FDX, auto, auto-flow
re0: Using defaults for TSO: 65518/35/2048
re0: Ethernet address: c0:3f:d5:93:3d:a7
re0: netmap queues/slots: TX 1/256, RX 1/256
pcib6: <ACPI PCI-PCI bridge> at device 28.3 on pci0
pcib6: [GIANT-LOCKED]
isab0: <PCI-ISA bridge> at device 31.0 on pci0
isa0: <ISA bus> on isab0
acpi_button0: <Power Button> on acpi0
acpi_button1: <Sleep Button> on acpi0
acpi_tz0: <Thermal Zone> on acpi0
orm0: <ISA Option ROM> at iomem 0xd0000-0xd0fff pnpid ORM0000 on isa0
atkbdc0: <Keyboard controller (i8042)> at port 0x60,0x64 on isa0
atkbd0: <AT Keyboard> irq 1 on atkbdc0
kbd0 at atkbd0
atkbd0: [GIANT-LOCKED]
est0: <Enhanced SpeedStep Frequency Control> on cpu0
Timecounters tick every 1.000 msec
TCP Hpts created 2 swi interrupt thread and bound 0
ugen0.1: <0x8086 XHCI root HUB> at usbus0
uhub0: <0x8086 XHCI root HUB, class 9/0, rev 3.00/1.00, addr 1> on usbus0
Trying to mount root from ufs:/dev/gpt/rootfs [rw,noatime]...
Root mount waiting for: CAM usbus0
ada0 at ahcich0 bus 0 scbus0 target 0 lun 0
ada0: <INTEL SSDMAEXC024G3H 9CV10379> ATA8-ACS SATA 2.x device
ada0: Serial Number CVHA3433003W024D
ada0: 300.000MB/s transfers (SATA 2.x, UDMA6, PIO 8192bytes)
ada0: Command Queueing enabled
ada0: 22902MB (46905264 512 byte sectors)
ada1 at ahcich1 bus 0 scbus1 target 0 lun 0
ada1: <INTEL SSDMAEXC024G3H 9CV10379> ATA8-ACS SATA 2.x device
ada1: Serial Number CVHA3433003R024D
ada1: 300.000MB/s transfers (SATA 2.x, UDMA6, PIO 8192bytes)
ada1: Command Queueing enabled
ada1: 22902MB (46905264 512 byte sectors)
GEOM: ada0: the secondary GPT header is not in the last LBA.
GEOM: diskid/DISK-CVHA3433003W024D: the secondary GPT header is not in the last LBA.
GEOM: ada1: the secondary GPT header is not in the last LBA.
GEOM: diskid/DISK-CVHA3433003R024D: the secondary GPT header is not in the last LBA.
uhub0: 7 ports with 7 removable, self powered
Root mount waiting for: usbus0
ugen0.2: <vendor 0x05e3 USB2.0 Hub> at usbus0
uhub1 on uhub0
uhub1: <vendor 0x05e3 USB2.0 Hub, class 9/0, rev 2.00/32.98, addr 1> on usbus0
uhub1: MTT enabled
uhub1: 4 ports with 4 removable, self powered
Root mount waiting for: usbus0
ugen0.3: <vendor 0x04d9 USB Keyboard> at usbus0
ukbd0 on uhub1
ukbd0: <vendor 0x04d9 USB Keyboard, class 0/0, rev 1.10/3.10, addr 2> on usbus0
kbd2 at ukbd0
mountroot: waiting for device /dev/gpt/rootfs...
Mounting filesystems...
tunefs: soft updates remains unchanged as enabled
GEOM: diskid/DISK-CVHA3433003W024D: the secondary GPT header is not in the last LBA.
tunefs: file system reloaded
tunefs: issue TRIM to the disk remains unchanged as enabled
GEOM: diskid/DISK-CVHA3433003W024D: the secondary GPT header is not in the last LBA.
tunefs: file system reloaded
** /dev/gpt/rootfs
FILE SYSTEM CLEAN; SKIPPING CHECKS
clean, 4768551 free (4487 frags, 595508 blocks, 0.1% fragmentation)
GEOM: diskid/DISK-CVHA3433003W024D: the secondary GPT header is not in the last LBA.
Setting hostuuid: 007547df-0bdc-e311-847d-c03fd5933da7.
Setting hostid: 0xa99c24ce.
Configuring vt: blanktime.
Configuring crash dump device: /dev/null
ddb: sysctl: debug.ddb.scripting.scripts: No such file or directory
.ELF ldconfig path: /lib /usr/lib /usr/local/lib /usr/local/lib/compat/pkg /usr/local/lib/compat/pkg /usr/local/lib/ipsec /usr/local/lib/perl5/5.32/mach/CORE
32-bit compatibility ldconfig path:
done.
>>> Invoking early script 'update'
>>> Invoking early script 'configd'
Starting configd.
>>> Invoking early script 'templates'
Generating configuration: OK
>>> Invoking early script 'backup'
>>> Invoking backup script 'captiveportal'
>>> Invoking backup script 'dhcpleases'
>>> Invoking backup script 'duid'
>>> Invoking backup script 'netflow'
>>> Invoking backup script 'rrd'
>>> Invoking early script 'carp'
CARP event system: OK
Launching the init system...done.
Initializing...........done.
em0: link state changed to UP
em3: link state changed to UP
re0: link state changed to DOWN
Starting device manager...
uhid0 on uhub1
uhid0: <vendor 0x04d9 USB Keyboard, class 0/0, rev 1.10/3.10, addr 2> on usbus0
done.
Configuring login behaviour...done.
Configuring loopback interface...
lo0: link state changed to UP
done.
Configuring kernel modules...
kernel trap 12 with interrupts disabled
kernel trap 12 with interrupts disabled
aesni0: No AES or SHA support.
done.
Setting up extended sysctls...done.
Setting timezone...done.
Writing firmware setting...done.
Writing trust files...done.
Setting hostname: gateway.webtool.space
Generating /etc/hosts...done.
Configuring system logging...done.
Configuring loopback interface...done.
Creating wireless clone interfaces...done.
Configuring LAN interface...
em3: link state changed to DOWN
done.
Configuring WAN interface...
em0: link state changed to DOWN
em3: link state changed to UP
em0: link state changed to UP
done.
Creating IPsec VTI instances...done.
Creating OpenVPN instances...
tun1: changing name to 'ovpns1'
done.
Generating /etc/resolv.conf...done.
Configuring firewall........done.
Starting PFLOG...done.
pflog0: promiscuous mode enabled
Configuring OpenSSH...done.
Starting web GUI...done.
Configuring CRON...done.
Setting up routes...done.
Generating /etc/hosts...done.
Starting DHCPv6 service...done.
Starting router advertisement service...done.
Setting up gateway monitors...done.
Configuring firewall........done.
Starting PFLOG...
pflog0: promiscuous mode disabled
done.
pflog0: promiscuous mode enabled
Syncing OpenVPN settings...
ovpns1: link state changed to UP
done.
Configuring dynamic DNS clients...done.
Starting NTP service...deferred.
Generating RRD graphs...done.
Configuring system logging...done.
>>> Invoking start script 'newwanip'
Reconfiguring IPv4 on em0:
pflog0: promiscuous mode disabled
pflog0: promiscuous mode enabled
OK
Reconfiguring IPv4 on ovpns1: OK
Reconfiguring IPv6 on em0:
pflog0: promiscuous mode disabled
pflog0: promiscuous mode enabled
OK
>>> Invoking start script 'freebsd'
WARNING: attempt to domain_add(netgraph) after domainfinalize()
setup em3
setup em0 [egress only]
Starting flowd_aggregate.
Starting flowd.
>>> Invoking start script 'syslog-ng'
Stopping syslog_ng.
Waiting for PIDS: 72205, 72205.
Starting syslog_ng.
>>> Invoking start script 'carp'
>>> Invoking start script 'cron'
Starting Cron: OK
>>> Invoking start script 'beep'
Root file system: /dev/gpt/rootfs
Tue Jan 26 20:06:48 PST 2021
```

Here's my `pciconf` output:

```
root@gateway:/var/run # pciconf -lv

hostb0@pci0:0:0:0:      class=0x060000 card=0x368817aa chip=0x0f008086 rev=0x0c hdr=0x00
    vendor     = 'Intel Corporation'
    device     = 'Atom Processor Z36xxx/Z37xxx Series SoC Transaction Register'
    class      = bridge
    subclass   = HOST-PCI
vgapci0@pci0:0:2:0:     class=0x030000 card=0x368817aa chip=0x0f318086 rev=0x0c hdr=0x00
    vendor     = 'Intel Corporation'
    device     = 'Atom Processor Z36xxx/Z37xxx Series Graphics & Display'
    class      = display
    subclass   = VGA
ahci0@pci0:0:19:0:      class=0x010601 card=0x368817aa chip=0x0f238086 rev=0x0c hdr=0x00
    vendor     = 'Intel Corporation'
    device     = 'Atom Processor E3800 Series SATA AHCI Controller'
    class      = mass storage
    subclass   = SATA
xhci0@pci0:0:20:0:      class=0x0c0330 card=0x368817aa chip=0x0f358086 rev=0x0c hdr=0x00
    vendor     = 'Intel Corporation'
    device     = 'Atom Processor Z36xxx/Z37xxx, Celeron N2000 Series USB xHCI'
    class      = serial bus
    subclass   = USB
none0@pci0:0:26:0:      class=0x108000 card=0x368817aa chip=0x0f188086 rev=0x0c hdr=0x00
    vendor     = 'Intel Corporation'
    device     = 'Atom Processor Z36xxx/Z37xxx Series Trusted Execution Engine'
    class      = encrypt/decrypt
pcib1@pci0:0:28:0:      class=0x060400 card=0x368817aa chip=0x0f488086 rev=0x0c hdr=0x01
    vendor     = 'Intel Corporation'
    device     = 'Atom Processor E3800 Series PCI Express Root Port 1'
    class      = bridge
    subclass   = PCI-PCI
pcib5@pci0:0:28:2:      class=0x060400 card=0x368817aa chip=0x0f4c8086 rev=0x0c hdr=0x01
    vendor     = 'Intel Corporation'
    device     = 'Atom Processor E3800 Series PCI Express Root Port 3'
    class      = bridge
    subclass   = PCI-PCI
pcib6@pci0:0:28:3:      class=0x060400 card=0x368817aa chip=0x0f4e8086 rev=0x0c hdr=0x01
    vendor     = 'Intel Corporation'
    device     = 'Atom Processor E3800 Series PCI Express Root Port 4'
    class      = bridge
    subclass   = PCI-PCI
isab0@pci0:0:31:0:      class=0x060100 card=0x368817aa chip=0x0f1c8086 rev=0x0c hdr=0x00
    vendor     = 'Intel Corporation'
    device     = 'Atom Processor Z36xxx/Z37xxx Series Power Control Unit'
    class      = bridge
    subclass   = PCI-ISA
none1@pci0:0:31:3:      class=0x0c0500 card=0x368817aa chip=0x0f128086 rev=0x0c hdr=0x00
    vendor     = 'Intel Corporation'
    device     = 'Atom Processor E3800 Series SMBus Controller'
    class      = serial bus
    subclass   = SMBus
pcib2@pci0:1:0:0:       class=0x060400 card=0x00000000 chip=0x8018111d rev=0x0e hdr=0x01
    vendor     = 'Microsemi / PMC / IDT'
    device     = 'PES12N3A 12-lane 3-Port PCI Express Switch'
    class      = bridge
    subclass   = PCI-PCI
pcib3@pci0:2:2:0:       class=0x060400 card=0x00000000 chip=0x8018111d rev=0x0e hdr=0x01
    vendor     = 'Microsemi / PMC / IDT'
    device     = 'PES12N3A 12-lane 3-Port PCI Express Switch'
    class      = bridge
    subclass   = PCI-PCI
pcib4@pci0:2:4:0:       class=0x060400 card=0x00000000 chip=0x8018111d rev=0x0e hdr=0x01
    vendor     = 'Microsemi / PMC / IDT'
    device     = 'PES12N3A 12-lane 3-Port PCI Express Switch'
    class      = bridge
    subclass   = PCI-PCI
em0@pci0:3:0:0: class=0x020000 card=0x704b103c chip=0x10bc8086 rev=0x06 hdr=0x00
    vendor     = 'Intel Corporation'
    device     = '82571EB/82571GB Gigabit Ethernet Controller (Copper)'
    class      = network
    subclass   = ethernet
em1@pci0:3:0:1: class=0x020000 card=0x704b103c chip=0x10bc8086 rev=0x06 hdr=0x00
    vendor     = 'Intel Corporation'
    device     = '82571EB/82571GB Gigabit Ethernet Controller (Copper)'
    class      = network
    subclass   = ethernet
em2@pci0:4:0:0: class=0x020000 card=0x704b103c chip=0x10bc8086 rev=0x06 hdr=0x00
    vendor     = 'Intel Corporation'
    device     = '82571EB/82571GB Gigabit Ethernet Controller (Copper)'
    class      = network
    subclass   = ethernet
em3@pci0:4:0:1: class=0x020000 card=0x704b103c chip=0x10bc8086 rev=0x06 hdr=0x00
    vendor     = 'Intel Corporation'
    device     = '82571EB/82571GB Gigabit Ethernet Controller (Copper)'
    class      = network
    subclass   = ethernet
re0@pci0:5:0:0: class=0x020000 card=0x368817aa chip=0x813610ec rev=0x05 hdr=0x00
    vendor     = 'Realtek Semiconductor Co., Ltd.'
    device     = 'RTL810xE PCI Express Fast Ethernet controller'
    class      = network
    subclass   = ethernet
``` 

Here's my `/boot/loader.conf` - I think it's pretty stock:

```
root@gateway:/var/run # cat /boot/loader.conf
##############################################################
# This file was auto-generated using the rc.loader facility. #
# In order to deploy a custom change to this installation,   #
# please use /boot/loader.conf.local as it is not rewritten, #
# or better yet use System: Settings: Tunables from the GUI. #
##############################################################

loader_brand="opnsense"
loader_logo="hourglass"
loader_menu_title=""

autoboot_delay="3"

# Vital modules that are not in FreeBSD's GENERIC
# configuration will be loaded on boot, which makes
# races with individual module's settings impossible.
carp_load="YES"
if_bridge_load="YES"
if_enc_load="YES"
if_gif_load="YES"
if_gre_load="YES"
if_lagg_load="YES"
if_tap_load="YES"
if_tun_load="YES"
if_vlan_load="YES"
pf_load="YES"
pflog_load="YES"
pfsync_load="YES"

# dynamically generated tunables settings follow
hw.ixl.enable_head_writeback="0"
net.enc.in.ipsec_bpf_mask="2"
net.enc.in.ipsec_filter_mask="2"
net.enc.out.ipsec_bpf_mask="1"
net.enc.out.ipsec_filter_mask="1"
net.inet.icmp.reply_from_interface="1"
net.local.dgram.maxdgram="8192"
debug.pfftpproxy=""
vfs.read_max="32"
net.inet.ip.portrange.first="1024"
net.inet.tcp.blackhole="2"
net.inet.udp.blackhole="1"
net.inet.ip.random_id="1"
net.inet.ip.sourceroute="0"
net.inet.ip.accept_sourceroute="0"
net.inet.icmp.drop_redirect="0"
net.inet.icmp.log_redirect="0"
net.inet.tcp.drop_synfin="1"
net.inet.ip.redirect="1"
net.inet6.ip6.redirect="1"
net.inet6.ip6.use_tempaddr="0"
net.inet6.ip6.prefer_tempaddr="0"
net.inet.tcp.syncookies="1"
net.inet.tcp.recvspace="65228"
net.inet.tcp.sendspace="65228"
net.inet.tcp.delayed_ack="0"
net.inet.udp.maxdgram="57344"
net.link.bridge.pfil_onlyip="0"
net.link.bridge.pfil_local_phys="0"
net.link.bridge.pfil_member="1"
net.link.bridge.pfil_bridge="0"
net.link.tap.user_open="1"
kern.randompid="347"
net.inet.ip.intr_queue_maxlen="1000"
hw.syscons.kbd_reboot="0"
net.inet.tcp.log_debug="0"
net.inet.icmp.icmplim="0"
net.inet.tcp.tso="1"
net.inet.udp.checksum="1"
kern.ipc.maxsockbuf="4262144"
vm.pmap.pti="1"
hw.ibrs_disable="0"
security.bsd.see_other_gids="0"
security.bsd.see_other_uids="0"

# dynamically generated console settings follow
#comconsole_speed
#boot_multicons
#boot_serial
#kern.vty
console="vidconsole"
```

Can't think of anything else to log ... hope that helps!

**Comment by fichtner** [comment]
Hi @averyfreeman,

My guess is that kernel.old shows 12.2-RELEASE-p2 so you have been using something not in OPNsense before.. the update to 20.7 brings you back to 12.1-RELEASE-p12 and it may cause your issues, although your dmesg also suggests 12.2-RELEASE-p2 is still in place which is odd to say the least...


Cheers,
Franco

**Comment by fichtner** [comment]
@averyfreeman do you have any news for me regarding this?
